Documentation/BuildInstructionsLadybird.md

```diff
@@ -136,7 +136,7 @@ ninja -C Build/ladybird debug
 To run without ninja rule:
 ```
 export SERENITY_SOURCE_DIR=$(realpath ../)
-./Build/ladybird/ladybird # or, in macOS: open ./Build/ladybird/ladybird.app
+./Build/ladybird/bin/ladybird # or, in macOS: open ./Build/ladybird/ladybird.app
```

Comment:
So when we mention the "commit category", we don't mean the PR description, but the actual commit message. Because Serenity uses a rebase workflow rather than a squash or merge workflow, we cherry-pick commits directly from PRs onto master. So for this change, what you should do is first change the path for macOS as well, and then force-push an amended commit

```suggestion
./Build/ladybird/bin/ladybird # or, in macOS: open ./Build/ladybird/bin/ladybird.app
```

```
git add Documentation
git commit --amend
# Edit commit message in your $EDITOR (or git config core.editor)
# To something like: Documentation: Update Ladybird build path in build instructions
# (fix is a dirty word around here :P it's not descriptive)
git push -f origin fix-path-to-ladybird-executable # or myfork or me or whatever you named your remote
```

Reply:
Thank you for clarifying :)